src/containers/edit-profile/professional-info-tab/ProfessionalInfoTab.styles.ts

```diff
@@ -1,4 +1,8 @@
-import { rootContainer } from '~/containers/edit-profile/common.style'
+import {
+  rootContainer,
+  updateProfileBtn,
+  innerContainer
+} from '~/containers/edit-profile/common.style'
```

Comment:
I suggest renaming `common.style` file to `common.styles` to be consistent because we mostly use `styles` word for styles